lib/ansible/module_utils/network/common/utils.py

```diff
@@ -42,7 +42,7 @@
 
 # Backwards compatibility for 3rd party modules
 from ansible.module_utils.common.network import (
-    to_bits, is_netmask, is_masklen, to_netmask, to_masklen, to_subnet, to_ipv6_network, VALID_MASKS
+    to_bits, is_netmask, is_masklen, to_netmask, to_masklen, to_subnet, to_ipv6_network, to_ipv6_subnet, VALID_MASKS
```

Comment:
I don't see why you're importing this to this module and using it from here. This line is just to help existing modules not fail imports when we moved these things out of here. I would rather see this change undone and all the imports below updated to reflect where the functions actually come from.

Reply:
Thank you for the advice! I updated my pull request.